Technolab.OnlineLibrary.Web/Views/Shared/_Layout.cshtml

```diff
@@ -27,20 +27,23 @@
                         </li>
                         @if (User?.Identity?.IsAuthenticated == true)
                         {
-                            <li class="nav-item">
-                                <a class="nav-link text-dark" asp-area="" asp-controller="Account">Account</a>
-                            </li>
-                            <li class="nav-item">
-                                <form method="post" asp-area="" asp-controller="Auth" asp-action="Logout">
-                                    <button type="submit" class="btn btn-link nav-link text-dark">Log Out</button>
-                                </form>
-                            </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-dark" asp-area="" asp-controller="Account">Account</a>
+                        </li>
+                        <li class="nav-item">

```

Comment:
This link should be available only when the users are logged in. I'd put it inside the Account view instead.

Reply:
I could try doing that but I wouldn't be sure how it looks as I still can't load css and js files into my browser locally (I know this isn't a very good justification) . Can we leave the link in the shared header for now? Seems like that's where all the navigation is.